src/main/services/features/breathingSpace/breathingSpaceService.ts

```diff
@@ -0,0 +1,41 @@
+import {getCaseDataFromStore, saveDraftClaim} from '../../../modules/draft-store/draftStoreService';
+import {BreathingSpace} from '../../../common/models/breathingSpace';
+import {ClaimDetails} from '../../../common/form/models/claim/details/claimDetails';
+
+const {Logger} = require('@hmcts/nodejs-logging');
+const logger = Logger.getLogger('interestService');
+
+const getBreathingSpace = async (claimId: string): Promise<BreathingSpace> => {
+  try {
+    const caseData = await getCaseDataFromStore(claimId);
+    return Object.assign(new BreathingSpace(), caseData.claimDetails?.breathingSpace);
```

Comment:
Approved but wanted to mention the below

This is only a preference, but I've been told that Object.assign is more of a hacky/forced way to do things so prefer Maro's way in `getClaimantResponse` where he checks if object exists, and if not, returns new instance of it
`return (claim.claimantResponse) ? claim.claimantResponse : new ClaimantResponse();`
```suggestion
    return (caseData.ClaimDetails?.breathingSpace) ? caseData.claimDetails?.breathingSpace : new BreathingSpace();
```

Reply:
Changes made, please have a look

Reply:
below does the same thing;
`return caseData.ClaimDetails?.breathingSpace ?? new BreathingSpace()`

Reply:
Updated. Can you have a look?